examples/simple-next-js-13/src/app/[[...slug]]/ClientComponent.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+import { BuilderComponent, builder } from '@builder.io/react';
+import builderConfig from '../../../builderConfig.json';
+
+builder.init(builderConfig.apiKey);
+
+export default function ClientComponent({ children, builderContent }: any) {
+  return (
+    <>
+      {children}
+      <br />
+      <p>This section is rendered client side</p>
```

Comment:
That's not accurate btw

Reply:
to elaborate, client components can be rendered on the server but will be rehydrated on the client while server components only run on the server